limits.md

```diff
@@ -1,17 +1,14 @@
 # Quotas and Limits
 
-In general, we currently allow
+In general, you are limited only by the amount of resources you can book within the system. What is inside of your cluster is your business.
 
-* an unlimited number of services
-* an unlimited number of components per service
+However, we suggest you contact us before going over the following limits:
 
-However, we enforce these restrictions:
+* 5 nodes per cluster
+* 32GB RAM per node
+* 50 pods per node
+* persistent storage volume over 100GB per volume
```

Comment:
Well, I would say "over 100GB" as only then they should tell us. But actually it can be "at"